Form/Type/VichImageType.php

```diff
@@ -19,6 +19,12 @@
  */
 class VichImageType extends VichFileType
 {
+    const STORAGE_RESOLVE_URI = 0;
```

Comment:
can we make constants private?

Reply:
Sure! I was not sure about the PHP supported version, but as it's 7.2 it's all good.

Reply:
Having them public allows to make the configuration quite readable :

```php
$builder->add('file', VichImageType::class, [
    'storage_resolve_method' => VichImageType::STORAGE_RESOLVE_PATH_ABSOLUTE,
]);
```

Where having them private would force to use integers or pre-defined strings for configuration. Not as nice IMO.